### PR TITLE
Add AI functions blog post

### PR DESCRIPTION
**Pull request by LauraDuRy**
## What does this PR do?

(Provide a description of what this PR does.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

(Write your answer here.)

### CODE REVIEW

File: static/images/blog/function-templates-overview.png
Comment (stnguyen90) [inline static/images/blog/function-templates-overview.png:1]: Please move the images in a folder matching the blog (`building-with-ai-function-templates`) so it's organized and we know what blog the image is for.

File: src/routes/blog/post/building-with-ai-function-templates/+page.markdoc [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+As we have covered in this blog, we have added a lot of great templates for you to get started on your AI journey. Including support for AI integrations:
+
+- [Pinecone](https://appwrite.io/docs/products/ai/integrations/pinecone)
+- [OpenAI](https://appwrite.io/docs/products/ai/integrations/openai)
+- [Hugging Face](https://appwrite.io/docs/products/ai/tutorials/image-classification)
+- [ElevenLabs](https://appwrite.io/docs/products/ai/integrations/elevenlabs)
+- [Perplexity](https://appwrite.io/docs/products/ai/integrations/perplexity)
+- [Replicate](https://appwrite.io/docs/products/ai/integrations/replicate)
+- [fal.ai](http://fal.aihttps://appwrite.io/docs/products/ai/integrations/fal-ai)
```
Comment (stnguyen90) [inline src/routes/blog/post/building-with-ai-function-templates/+page.markdoc:186]: broken link

```suggestion
- [fal.ai](https://appwrite.io/docs/products/ai/integrations/fal-ai)
```

**Review by stnguyen90 — CHANGES_REQUESTED** [review]

**Review by stnguyen90 — CHANGES_REQUESTED** [review]
Images still need to be moved.

File: static/images/blog/function-templates-overview.png
Comment (LauraDuRy) [reply static/images/blog/function-templates-overview.png:1]: @adityaoberai can you help me with this? or show me how to do this.

**Comment by adityaoberai** [comment]
> Images still need to be moved.

@stnguyen90 shifted

**Review by stnguyen90 — APPROVED** [review]

**Pull request merged**